.github/workflows/go.yml

```diff
@@ -92,8 +92,8 @@ jobs:
             fail-fast: false
             matrix:
                 go:
-                    - 1.18.10
```

Comment:
why the indentation change? it doesn't match the rest of the doc

Reply:
That was unintentional, thanks for catching this